packages/application-components/package.json

```diff
@@ -31,7 +31,6 @@
     "prepare": "./../../scripts/version.js replace"
   },
   "dependencies": {
-    "@apollo/client": "3.3.8",
```

Comment:
FYI: in the other packages where we do use it, it's declared as a peer dependency